tests/test_pipelines.py

```diff
@@ -15,6 +15,7 @@
 FEATURE_EXTRACT_FINETUNED_MODELS = ["sshleifer/tiny-distilbert-base-cased"]
 TEXT_CLASSIF_FINETUNED_MODELS = ["sshleifer/tiny-distilbert-base-uncased-finetuned-sst-2-english"]
 TEXT_GENERATION_FINETUNED_MODELS = ["sshleifer/tiny-ctrl"]
+ZERO_SHOT_CLASSIFICATION_FINETUNED_MODELS = ["huggingface/distilbert-base-uncased-finetuned-mnli"]
```

Comment:
@sshleifer This is the smallest MNLI model I could find. If you don't like it, in theory I could use any classifier since I'm only testing input validity, it would just be a bit weird.

Reply:
I would suggest either:
(a) let it be weird and just test shapes. Use tiny random classifier.
(b) write a stronger @slow test that checks answers.
(c) both

I think C is the best because I estimate it will take you 20 minutes and save you 2 GH issues a months for 6 months.

Reply:
Done